### PR TITLE
Updating libcontainer deps

### PR DESCRIPTION
**Pull request by vishh**
fixes https://github.com/kubernetes/kubernetes/issues/28577#issuecomment-231800140

### CODE REVIEW

**Comment by vishh** [comment]
Depends on #1369

**Comment by vishh** [comment]
ok to test

**Comment by vishh** [comment]
cc @timstclair

**Comment by vishh** [comment]
@k8s-bot test this

**Comment by vishh** [comment]
@timstclair This patch is ready for review now.

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -327,78 +327,83 @@
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer",
-			"Comment": "v0.0.7",
-			"Rev": "7ca2aa4873aea7cb4265b1726acb24b90d8726c6"
+			"Comment": "v1.0.0-rc1-57-g4eb8c2f",
```
Comment (timstclair) [inline Godeps/Godeps.json]: Do we want to update to a patched release instead of head?

**Pull request closed without merging**